src/Common/Auth/AuthUtils.php

```diff
@@ -642,6 +642,7 @@ public function updatePassword($activeUser, $targetUser, &$currentPwd, &$newPwd,
             $updateParams = array();
             $updateSQL = "UPDATE `users_secure`";
             $updateSQL .= " SET `last_update_password` = NOW()";
+            $updateSQL .= ", `login_fail_counter` = 0";
```

Comment:
Should be set to our default as if a new user right?

Reply:
this is in the update user block

Reply:
@stephenwaite is correct, this is part of the update user section.